assets/js/updaters.js

```diff
@@ -753,7 +745,21 @@ function updateFissure() {
           }
         });
         if (!filtered) {
-          $('#fissurebody').before(fissureRow);
+          let added = false;
+
+          // Order by tier number
+          $('#fissureList .fissure').each((index, existingFissure) => {
+            if ($(existingFissure).attr('data-tiernum') > fissure.tierNum) {
```

Comment:
Could you grab it's index in the list of existing fissures, then using that, insert it before the next I'd if it exists, and if not, then add it at the end?

Reply:
I'm not sure what you mean by that, sorry. What indices and values would we be storing here? Could you provide some pseudocode?

Reply:
Not really, I'm on a phone. We'd be using the IDs of the fissure rows, so finding what should be before or after it and inserting as applicable

Reply:
Still don't know what you mean.  Please discuss with me on Discord when you have time, I guess.

Reply:
This is fine for the current solution. I think it's actually just as efficient as what i was envisioning.